docker-compose.yml

```diff
@@ -1,8 +1,10 @@
 db:
   build: db
```

Comment:
with newer version of compose, it is fine:
https://docs.docker.com/compose/compose-file/#build